Firmware/ultralcd.cpp

```diff
@@ -3340,27 +3340,15 @@ static void lcd_sort_type_set() {
 #ifdef TMC2130
 static void lcd_crash_mode_info()
 {
-	lcd_update_enable(true);
```

Comment:
Note: This function is always called as a “menu” and with all menus the LCD updates are always enabled. The `lcd_update_enable()` here is redundant/not useful.